### DISCUSSION TITLE
Configure RSS-Feed (e.g. exclude private pages)

### DISCUSSION THREAD

**Discussion by can-canned** [automadcms/discussions#20] [Q&A]
Hi,
is there already a config variable in the rss-feed to ignore hidden pages?
Thanks in advance for tips!
Cheers
Pascal

**Comment by can-canned** [comment]
Comment from the old Automad-forum (onechrisjones, Apr 26, 2022):
"It looks like the RSS feed currently just syndicates any additions made to the whole site... My subscribers probably don't want to know when I add a page to my site though. It looks like hidden pages don't make it to the feed... but I think the RSS feed should only have items with a date. I think that might be an easy and logical way to enhance the RSS feature."

In which file or via which function is the RSS feed currently generated (Automad version 1)? Could not find the file/function yet.

**Comment by marcantondahmen** [comment]
Hi, hidden pages are excluded by default. In version 1, the `Core\Feed` class is responsible for generating the feed.
